### ISSUE TITLE
Investigate How Low Temperatures Affect the Sensors

### ISSUE THREAD

**Issue by eaudiffred** [Ribbit-Network/ribbit-network-frog-sensor#129]
There is a potential concern with low outside temperatures causing problems with sensor accuracy or even the ability to operate.
Below is a table of recommended operating temperatures for each component.  These numbers were taken from the data sheet for each component.  
![image](https://user-images.githubusercontent.com/88562047/179015408-1b987b4e-684d-4eb5-a3e4-41f061dea5ca.png)

**Comment by eaudiffred** [comment]
https://www.adafruit.com/product/5187#technical-details
This sensor has a wider operating temp -10C to 60C which is better, but still not great.  I'd like to see the low end somewhere around -20C or -30C.  
It uses slightly less power and is also $10 cheaper than the SCD-30
The accuracy is slightly worse though.  5% compared to the SCD-30 at 3%
The range is also significantly less.  400ppm-2,000ppm vs 400ppm-10,000ppm on the SCD-30.  I don't think the range should be an issue though.

**Comment by keenanjohnson** [comment]
Hey @eaudiffred ! Yeah we had seen that sensor, but decided to go with the SCD-30 as it has a slightly more accurate range. However, given that we have now seen temperature range issues with the SCD-30, it is probably worth digging into further.

**Comment by eaudiffred** [comment]
It's starting to get cold here in Utah and I notice that my sensor is going in and out.  It doesn't perfectly correlate to the temperature swings each day, but I'm wondering if the cold is causing the problem.  

I'll try rebooting today.  It's only been below freezing a few times so far, so I'm surprised to see the sensor dropping out as much as it is.  

![image](https://user-images.githubusercontent.com/88562047/200069042-05858bea-0b1e-4fed-a92e-9246d1482018.png)
![image](https://user-images.githubusercontent.com/88562047/200069197-1230471e-3c06-4239-909c-41d48d057e85.png)

**Comment by eaudiffred** [comment]
As soon as I can get a V4 sensor up and running I'll be interested to see if our low temp issues are resolved.  I believe the ESP32 has a lower temp operating range than the Raspberry Pi.

**Comment by keenanjohnson** [comment]
yes agreed! We are still finishing out the setup software for the V4 sensor, but this will be quite interesting to test! /milestone/1

**Comment by eaudiffred** [comment]
We are now running V4 of the Frog sensor.  I have one outside my home and will monitor this winter.
